### PR TITLE
[TC-8527] [TC-9086] Add load carriers

### PR DESCRIPTION
**Pull request by marcmatt**
## Description

https://tradecloud.atlassian.net/browse/TC-8527 story
https://tradecloud.atlassian.net/browse/TC-9086 sub task

Please review https://app.gitbook.com/o/-LNyIUZSvpZWZ81yEr7V/s/-LNyJ9UuwLCjTvA2sqQR-887967055/~/revisions/YwgnGvhasacQsZBElFKE/processes/shipment/buyer/receive

### Scope
This PR adds load carriers to the shipment API manual

### Submitter pre-flight check

- [x] Review your documentation
- [x] Add labels `needs reviewing`
- [x] Assign PR and JIRA ticket to 'Reviewer' (in 'Reviewing' state)

### Reviewer pre-flight check

- [ ] Review documentation in PR
- [ ] Remove label `needs reviewing`
- [ ] Merge the PR and remove the release for this branch from Gitbook
- [ ] Set Jira ticket to 'Released'

### CODE REVIEW

**Comment by marcmatt** [comment]
@roy-tc @Sjorstradecloud Please review 
Please review https://app.gitbook.com/o/-LNyIUZSvpZWZ81yEr7V/s/-LNyJ9UuwLCjTvA2sqQR-887967055/~/revisions/EG6qVQ4HpXwrXUiNvEJk/processes/shipment/buyer/receive

File: shipment/buyer/receive/README.md [hunk 1]
```diff
@@ -30,7 +35,8 @@ The shipment module is under development. The API and documentation may change.
 * `trackingNumber`: the tracking number of the shipment as provided by the carrier or courier
 * `supplierShipment`: the supplier side header of the shipment, see [Supplier shipment header](#buyer-shipment-header)
 * `buyerShipment`: the buyer side header part of the shipment, see [Buyer shipment header](#buyer-shipment-header)
-* `lines`: a list of all the shipment lines, see [Shipment line](#shipment-line)
+* `loadCarriers`: a list of all the load carriers in this shipment, each load carrier containing shipment lines, see [Load carrier](#load-carrier)
+* `lines`: a list of all the shipment lines, not loaded in a load carrier, see [Shipment line](#shipment-line)
 * `locations`: the departure, next destination and final destination locations of a shipment, see [Shipment locations](#shipment-locations)
 * `meta`: meta information about the shipment, see [Shipment meta information](#shipment-meta-information)
```
Comment (Sjorstradecloud) [inline shipment/buyer/receive/README.md:42]: Supplier shipment header is out dated, missing invoice number, contacts, supplierParty
buyer party also missing buyerparty, contacts, purchaseOrderTerms

File: shipment/buyer/receive/README.md [hunk 2]
```diff
@@ -30,7 +35,8 @@ The shipment module is under development. The API and documentation may change.
 * `trackingNumber`: the tracking number of the shipment as provided by the carrier or courier
```
Comment (Sjorstradecloud) [inline shipment/buyer/receive/README.md:35]: Missing the transportMode & identifiers

**Review by Sjorstradecloud — COMMENTED** [review]
Loadcarrier part looks good, however missing a lot of other things we added/ changed, which might be in another PR but just mentioned them here to be sure.

**Comment by marcmatt** [comment]
@Sjorstradecloud Ok, I will update the API manual to be consistent with the API specs.

File: shipment/buyer/receive/README.md [hunk 3]
```diff
@@ -30,7 +35,8 @@ The shipment module is under development. The API and documentation may change.
...
 * `supplierShipment`: the supplier side header of the shipment, see [Supplier shipment header](#buyer-shipment-header)
 * `buyerShipment`: the buyer side header part of the shipment, see [Buyer shipment header](#buyer-shipment-header)
-* `lines`: a list of all the shipment lines, see [Shipment line](#shipment-line)
+* `loadCarriers`: a list of all the load carriers in this shipment, each load carrier containing shipment lines, see [Load carrier](#load-carrier)
+* `lines`: a list of all the shipment lines, not loaded in a load carrier, see [Shipment line](#shipment-line)
 * `locations`: the departure, next destination and final destination locations of a shipment, see [Shipment locations](#shipment-locations)
```
[the remark this answers is not in the document]
Comment (marcmatt) [reply shipment/buyer/receive/README.md:43]: Fixed

File: shipment/buyer/receive/README.md (hunk 1 above)
Comment (marcmatt) [reply shipment/buyer/receive/README.md:45]: Fixed

**Comment by marcmatt** [comment]
@Sjorstradecloud Please review https://app.gitbook.com/o/-LNyIUZSvpZWZ81yEr7V/s/-LNyJ9UuwLCjTvA2sqQR-887967055/~/revisions/nodvaKIaoJ0queJk8Ttj/processes/shipment/buyer/receive

**Review by Sjorstradecloud — COMMENTED** [review]
Look good, left 1 comment.

**Comment by marcmatt** [comment]
@Sjorstradecloud Please review https://app.gitbook.com/o/-LNyIUZSvpZWZ81yEr7V/s/-LNyJ9UuwLCjTvA2sqQR-887967055/~/revisions/YwgnGvhasacQsZBElFKE/processes/shipment/buyer/receive

**Review by Sjorstradecloud — APPROVED** [review]
Looks good!

**Pull request merged**